src/init/features/apphosting/repo.ts

```diff
@@ -135,7 +135,7 @@ async function promptRepositoryUri(
 ): Promise<{ remoteUri: string; connection: gcb.Connection }> {
```

Comment:
I think location in the parameters can be deleted now, I don't think its being used